### DISCUSSION TITLE
Find Driver for main Def.

### DISCUSSION THREAD

**Discussion by mbxymail** [cgtinker/BlendArMocap#147] [General]
In the tutorial you show how to adjust an CGT driver for left hand. The feet on my rig are close and I woiuld like to add distance, How do I frind drivers for  main iks. THaks

**Comment by cgtinker** [comment]
I think feet and hands currently use `ik chain-drivers` which don't have a preset offset parameter. However, it's actually just a mask to help setting up drivers - therefore, you can change the actual drivers manually.

For any object which contains data that gets mapped (like the `foot_index.L`, `wrist.L` etc) there is a corresponding driver object which has the same name with a '.D' suffix (like `foot_index.L.D`, `wrist.L.D` etc). You can modify the driver properties in the `drivers editors`.

**Reply by mbxymail** [reply]
I tried a different video and it seem to work better.  I have a question on the rig:foot_ik_L, 
I noticed you moved the hand_ik in one of you tutorial, but I cann't seem to move the foot_ik_L

 BUT I AM IMPRESSED. I THINK A LITTLE MORE WORK ON FOOT and STABLIZED the feet position (suggestion add an empty to bind all point as a refernce to the empty)
 
 Plus rig seem to lean forward too much.

**Comment by Kazeem6511** [comment]
I don't know about programming especially writing command at all please could someone guide me am just confuse right now
